multi-datacenter/docker-compose.yml

```diff
@@ -115,6 +115,7 @@ services:
     ports:
       - "8382:8382"
       - "9892:9892"
+    command: "bash -c 'cp /usr/share/java/kafka-connect-replicator/replicator-rest-extension-5.4.0-SNAPSHOT.jar /etc/kafka-connect/jars/; /etc/confluent/docker/run'"
```

Comment:
Hard-coding this to `5.4.0-SNAPSHOT` will require manual edit for each release bump.  Instead, can you please modify this to catch any release?  something along the lines of `...replicator-rest-extension-*.jar...`

cc: @rspurgeon

Reply:
Done